_includes/_navigation.html

```diff
@@ -60,7 +60,7 @@ <h1 class="show-for-small-only"><a href="{{ site.url }}{{ site.baseurl }}/{{ sit
           {% endif %}
 
           <li class="divider"></li>
-          <li{% if page.lang == lang.short %} class="active"{% endif %}><a href="{{ site.url }}{{ site.baseurl }}{{ pageurl }}">{{ lang.short | escape }}</a></li>
```

Comment:
This file should not have been edited (it is being reverted to a previous version here.)